packages/nimble-components/src/select/tests/select.foundation.spec.ts

```diff
@@ -245,24 +245,6 @@ describe('Select', () => {
         await disconnect();
     });
 
```

Comment:
It looks like you can now select a disabled option using the keyboard. This seems wrong.

Reply:
https://github.com/ni/nimble/assets/10500124/0b097f5d-dde2-4d94-b45b-2dfae04ee550

Reply:
Whoops! Fixed and added test.